#DPATCHLEVEL=0

2005-11-28  Sergio Lopez  <koro@sinrega.org>

	* linux/src/drivers/net/apricot.c (apricot_probe): Increase eth_addr
	size to 8 so we do not trash the stack when reading from the io ports.

--- linux/src/drivers/net/apricot.c	2005-11-28 00:13:12.000000000 +0100
+++ linux/src/drivers/net/apricot.c	2005-11-28 00:13:05.000000000 +0100
@@ -681,7 +681,7 @@
     struct i596_private *lp;
     int checksum = 0;
     int ioaddr = 0x300;
-    char eth_addr[6];
+    char eth_addr[8];
     
     /* this is easy the ethernet interface can only be at 0x300 */
     /* first check nothing is already registered here */
